### ISSUE TITLE
Youtube will start scrolling automatically (and infinitely) if you scroll down too fast on the homepage

### ISSUE THREAD

**Issue by Brave-Matt** [brave/brave-browser#31172]
## Description 
When you land on Youtube.com, if you start scrolling down the video feed too quickly before the first videos are fully loaded (it seems), the page begins to auto-scroll to the bottom of the page on its own very quickly and will continue to do this infinitely until you interact with the screen again

## Steps to reproduce 

   1. Visit youtube.com
   2. As the page loads, begin scrolling until you hit the bottom of the page (which typically triggers the page to start loading additional content)
   3. Observe the site go crazy and begin to infinitely "scroll and load"

## Actual result 

https://github.com/brave/brave-browser/assets/3299098/b07e993f-6dae-498d-9958-4430cf3cbd2b

## Expected result
For that not to happen

## Issue reproduces how often 
Easily

## Version/Channel Information:

- Can you reproduce this issue with the current Play Store version? Yes
- Can you reproduce this issue with the current Play Store Beta version? Yes 
- Can you reproduce this issue with the current Play Store Nightly version? Yes

## Device details
   - Install type (ARM, x86):
   - Device type (Phone, Tablet, Phablet): Phone
   - Android version: Android 13
   
## Brave version
1.52.126

### Website problems only
- Does the issue resolve itself when disabling Brave Shields? No
- Does the issue resolve itself when disabling Brave Rewards? No
- Is the issue reproducible on the latest version of Chrome? No

**Comment by saksham-81** [comment]
Any update on this?

**Comment by dannyparsons** [comment]
Has this issue been addressed? I am also facing this exact issue on YouTube with Brave on Android.

**Comment by ryanbr** [comment]
Given the age of the issue, surprised its still happening. Any added filter lists in brave://settings/shields content filtering?

Which version of Android/Device/ and Brave version @dannyparsons

**Comment by dannyparsons** [comment]
Thanks for replying. 

No added filters. I also tried with Shield Down and get the same behaviour.

Android 14
Build Number FP5.UT2K.B.098.20250109
Fairphone 5 device 

Brave 1.75.175, Chromium 133.0.6943.54
Android 14; Build/UKQ1.230924.001

**Comment by ryanbr** [comment]
We saw reports that it occurs in other browsers (like Chrome) is that also the case?

**Comment by dannyparsons** [comment]
I just tried on Chrome and do get the same issue. Does this mean it's likely an issue with Chromium?

**Comment by ryanbr** [comment]
Yeah, not sure if fixable. But you could try: in brave settings/shields/custom rules.

`m.youtube.com##body,html:style(position: unset !important)`
or
`m.youtube.com##+js(rc, watch-scroll, html, stay)`
or
`m.youtube.com##body:style(overflow: scroll !important)`

**Comment by dannyparsons** [comment]
Thanks for the suggestions. 

I went to `Settings > Brave Shields and Privacy > Content Filtering > Create custom filter`, and then added these each on a separate line. Is that the correct place? 

If so, I still have the same behaviour unfortunately.

**Comment by ryanbr** [comment]
Was worth a try, something that needs to be fixed upstream

**Comment by dannyparsons** [comment]
Ok thanks. Is there a way to report this further up?

**Comment by ntzolov** [comment]
any fixes?

**Comment by Brave-Matt** [comment]
Another +1 from Community:
https://community.brave.com/t/youtube-browsing-goes-into-loading-loop-when-it-loads-explore-more-options-banner/631659/2

**Comment by Uni-verse** [comment]
@Brave-Matt 

I was able to reproduce this easily on my Samsung device, `Android 14` using current GPS version`1.80.120`, I am not seeing anything obvious in the JS console. 

https://github.com/user-attachments/assets/7e67771f-a274-4fb4-88be-112a3502263e

<img width="1917" height="1127" alt="Image" src="https://github.com/user-attachments/assets/33544141-d592-4644-ad2d-ec4730e9a85e" />

cc: @ryanbr @timchilds

**Comment by kjozwiak** [comment]
Removed the Android 13 label as I'm running into the above on my S25 Ultra that's running Android 15 using the following build(s):

```
Brave 1.84.11 Chromium: 140.0.7339.41 (Official Build) canary (64-bit) 
--- | ---
Revision | 6882a5b68015c3520573108f03b6238be81c9b36
OS | Android 15; Build/AP3A.240905.015.A2; 35; REL
```

https://github.com/user-attachments/assets/7363ee1d-262c-4ba5-845a-27bb52005156

**Comment by Brave-Matt** [comment]
Still an issue -- I know that it has the Chromium upstream tag but I'm wondering if there's any way we can work on fixing this ourselves. Its a super annoying issue. 

Another user report of this here: https://community.brave.app/t/fix-if-youtube-on-brave-gets-stuck-autoscrolling-android-mobile-only/647826

@timchilds @SergeyZhukovsky

**Comment by timchilds** [comment]
@simoarpe one for you when you next investigate YT improvements.

**Comment by Sp0kzz** [comment]
still happening in May 2026, i don't think it's a chromium issue more than a Youtube one and they don't care as they want us to use their app. Many features are left out on purpose as well like the livechat and the messed up Portrait videos.
